kubernetes/samples/scripts/create-fmw-infrastructure-domain/domain-home-on-pv/wdt/create-domain-script.sh

```diff
@@ -74,7 +74,7 @@ WDT_VAR_FILE=${WDT_VAR_FILE:-"$SCRIPTPATH/create-domain-inputs.yaml"}
 WDT_DIR=${WDT_DIR:-/shared/wdt}
 
```

Comment:
I just noticed that I missed to add `WDT_VERSION=${WDT_VERSION:-1.4.0}` in this file. Can you please add?